pages/print_destination_plate_labels.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <b-container>
+    <b-row>
+      <b-col>
+        <h1>Print destination plate labels</h1>
+        <p class="lead"></p>
+        <p>
+          <b-button
+            id="printLabels"
+            variant="success"
+            :disabled="isBusy"
+            @click="printLabels"
+          >
+            Create report
+            <b-spinner v-show="isBusy" small></b-spinner>
+          </b-button>
+        </p>
+        <!-- TODO: better in a component of its own? -->
```

Comment:
Set status will be used here I just haven't set it up yet. Maybe you could help me with this? Maybe we could pair on it? Sounds like you have some good ideas. Don't think it would take too long.